rle_array/autoconversion.py

```diff
@@ -36,24 +37,38 @@ def _uses_datetimeblock(dtype: Union[np.dtype, ExtensionDtype]) -> bool:
     return issubclass(vtype, np.datetime64)
 
 
-def auto_convert_to_rle(df: pd.DataFrame, threshold: float = 1.0) -> pd.DataFrame:
+def auto_convert_to_rle(
+    df: pd.DataFrame, threshold: Optional[float] = None
+) -> pd.DataFrame:
     """
     Auto-convert given DataFrame to RLE compressed DataFrame.
 
     .. important::
 
         Datetime columns are currently not compressed due to pandas not supporting them.
 
+    Please note that RLE can, under some circumstances, require MORE memory than the uncompressed data. It is not
```

Comment:
It should also not be None?

Reply:
It can, in which case no limit is applied. See parameter description. (or maybe I misinterpret your question).

Reply:
I thought i removed that comment 😓 I saw that too